cg/cli/clean.py

```diff
@@ -129,6 +129,7 @@ def mip_run_dir(context, yes, case_id, dry_run: bool = False):
     try:
         if mip_analysis_api.is_latest_analysis_ongoing(case_id):
             LOG.warning(f"Analysis for case {case_id} is still ongoing!")
+            return
```

Comment:
Since we return here now I think we should tell the user what to do to be able to clean